crates/eth-rpc/src/api/eth_api.rs

```diff
@@ -7,106 +7,106 @@ use reth_rpc_types::{
     TransactionReceipt, TransactionRequest, Work,
 };
 
-#[rpc(server)]
+#[rpc(server, namespace = "eth")]
 #[async_trait]
 pub trait EthApi {
-    #[method(name = "eth_blockNumber")]
+    #[method(name = "blockNumber")]
     async fn block_number(&self) -> Result<U64>;
 
     /// Returns the protocol version encoded as a string.
     #[method(name = "net_version")]
```

Comment:
what about this `net_` method? will it become `eth_net_version` ? now that we've split things in namespaces, we may just add a `net` namespace?

Reply:
just saw that it's what you did! but then we need to remove this right?

Reply:
oh! I thought I have removed it from here as well! Thanks for the catch.